packages/material-ui-lab/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -83,6 +83,10 @@ export interface AutocompleteProps<T>
    * Force the visibility display of the popup icon.
    */
   forcePopupIcon?: true | false | 'auto';
+  /**
+   * The numbers of items that will be visible
+   */
+  filterMax?: number;
```

Comment:
What do you think of this name for the prop?

```suggestion
  filterMaxTags?: number;
```

Reply:
I liked, it's better.